core/src/main/scala/chisel3/properties/Class.scala

```diff
@@ -127,6 +127,7 @@ case class ClassType private[chisel3] (name: String) { self =>
         override def convertUnderlying(value: Property[ClassType] with self.Type) = value.ref
       }
   }
+  def copy(name: String = this.name) = new ClassType(name)
```

Comment:
I think we want this as private

Reply:
afaict these need to be public for scala3..